lib/active_merchant/billing/gateways/trans_first_transaction_express.rb

```diff
@@ -399,7 +499,7 @@ def message_from(succeeded, response)
 
           message = RESPONSE_MESSAGES[code]
           extended = EXTENDED_RESPONSE_MESSAGES[extended_code]
-          ach_response = response['achResponse']
+          ach_response = response['achResponse'].try(:[], 'Message') || response['achResponse']
```

Comment:
This is a bit different then: https://github.com/rednovalabs/active_merchant/pull/9/commits/a8d0b6981fe03806409aab36cf13cb7464f8d4a2